target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,322 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+	model = "ZTE MF286C3";
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+		bootargs-override = "console=ttyS0,57600n8";
+	};
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_pins>;
+
+		led-6  { // p0_0 sig00
+			color = <LED_COLOR_ID_BLUE>;
```

Comment:
1. It's better to resort nodes to make them start from led-0. Or just rename node names.

2. Please use C language comment style: `/* comment */`.

3. LED `function` property is necessary.